odoo/addons/base/models/res_company.py

```diff
@@ -309,6 +312,8 @@ def create(self, vals_list):
         # Make sure that the selected currencies are enabled
         companies.currency_id.sudo().filtered(lambda c: not c.active).active = True
 
+        companies.sudo().install_l10n_modules()
```

Comment:
Why sudo?

Reply:
There chances of getting a trace back, better to use `sudo` most of the time when you are override `create`, better to use `sudo` when calling a method if require which can skip the access errors and trace back

Reply:
Why don't we use sudo everywhere then?